lib/babl.rb

```diff
@@ -37,38 +37,41 @@ def self.version
     `#{bin_path} -version`.strip
   end
 
-  def self.client endpoint
-    @clients ||= {}
-    @clients[endpoint || ENV['BABL_ENDPOINT'] || 'default'] ||= begin
-      path = bin_path
-      path = "#{path} -endpoint #{endpoint}" if endpoint
-      Quartz::Client.new(bin_path: path)
-    end
+  def self.client
+    @client ||= Quartz::Client.new(bin_path: bin_path)
   end
 
   def self.module! name, opts = {}
+    res = call! name, opts
```

Comment:
`call!` gives raw response here, and we process it as before. Fetching the payload coming soon.